pkgs/applications/graphics/tev/default.nix

```diff
@@ -49,6 +49,7 @@ stdenv.mkDerivation rec {
     license = licenses.bsd3;
     platforms = platforms.unix;
     badPlatforms = [ "aarch64-linux" ]; # fails on Hydra since forever
+    broken = stdenv.isDarwin; # builds need work
```

Comment:
the comment should at least indicate what part and a short extract from the error. Right now it is not very helpfull.